policy/modules/system/systemd.te

```diff
@@ -1461,6 +1462,7 @@ optional_policy(`
 	dbus_system_bus_client(systemd_resolved_t)
 	dbus_watch_system_bus_runtime_dirs(systemd_resolved_t)
 	dbus_watch_system_bus_runtime_named_sockets(systemd_resolved_t)
+	sysnet_dbus_chat_dhcpc(systemd_resolved_t)
```

Comment:
Dhcpc does not appear to have the permissions to be a dbus service. This should probably be a call to `sysnet_dns_name_resolve` in dhcpc instead. That includes the dbus communication to systemd-resolved.

Reply:
Thanks. It works when using sysnet_dns_name_resolve instead of sysnet_dbus_chat_dhcpc

Reply:
> Thanks. It works when using sysnet_dns_name_resolve instead of sysnet_dbus_chat_dhcpc

I'm confused. On your most recent push you restored the `sysnet_dbus_chat_dhcpc` was this an accident?